test/test_nn.py

```diff
@@ -2606,6 +2607,61 @@ def test_orthogonal(self):
                                          torch.eye(rows) * gain ** 2, prec=1e-6)
 
 
+# functional loss wrappers
+
+
+class _SmoothL1LossFunctional(_Loss):
```

Comment:
Depends on the implementations, I think? Kind of funny though if we have a situation hypothetically where one implementation wraps the other, and both tests simply compare the Module result with the functional result :-)